pkg/reconciler/pipelinerun/cancel.go

```diff
@@ -122,6 +127,45 @@ func cancelPipelineTaskRuns(ctx context.Context, logger *zap.SugaredLogger, pr *
 	return errs
 }
 
+// getChildObjectsToCancel looks in the PipelineRunStatus for TaskRun and Run names to cancel. It looks in
```

Comment:
nit: this function comment describes the implementation, and the name describes how the function will be used.
I'd prefer something like:
// getChildObjects returns taskruns and runs owned by the pipelineRun, based on the value of the embedded status flag.

Reply:
👍 I'm renaming it to `filterChildObjects`.